README.md

```diff
@@ -1,19 +1,31 @@
 # data-and-algorithms
 Let's go over data structures and algorithms together! 
-We will use Cracking the Coding Interview as reference.
+We will use [Cracking the Coding Interview Book](http://www.crackingthecodinginterview.com/) as reference.
 
 Feel free to add your solutions in your channel of your choice 😃
```

Comment:
now that you're editing this, can you change `channel` for `programming language`?